ontology/yaml/resources/SAFETY/entity_types/ABSTRACT.yaml

```diff
@@ -94,3 +94,13 @@ LM:
   uses:
   - level_status 
 
+VA6X:
```

Comment:
You should not be mixing types of vibration being detected on the same type. These look like distinct types to me: one for the elevators, one for heaters, one for the gas_heaters. Furthermore, it seems like these are alarms that belong on existing types (e.g., the elevator alarms should be included on an existing `ELV` type, not modeled as a separate vibration sensor type.

Reply:
okay, these can potentially be modelled as 3 different sensors `SENSOR_LV3A`, i can't add them to the devices at the moment